<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DRWidget</class>
 <widget class="QWidget" name="DRWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>154</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Add new record</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0">
      <item row="1" column="4">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
        <item>
         <widget class="QtySpinInput" name="spin_qty"/>
        </item>
        <item>
         <widget class="QLabel" name="label_unit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="label_reason2">
        <property name="text">
         <string>Optional</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_qty">
        <property name="text">
         <string>Quantity</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_date">
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="TextInput" name="edit_reason_a">
        <property name="placeholderText">
         <string>reason</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_reason">
        <property name="text">
         <string>Reason</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_products">
        <property name="text">
         <string>Product</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="TextInput" name="edit_reason_b">
        <property name="placeholderText">
         <string>additional reason</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="DateInput" name="edit_date">
        <property name="placeholderText">
         <string>distribute date</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ExpandableComboBox" name="combo_batch">
        <property name="insertPolicy">
         <enum>QComboBox::NoInsert</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="action_addnext">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Insert</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="action_addexit">
       <property name="text">
        <string>Insert and &amp;Exit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="action_update">
       <property name="text">
        <string>&amp;Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="action_clear">
       <property name="text">
        <string>C&amp;lear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="action_cancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextInput</class>
   <extends>QLineEdit</extends>
   <header>TextInput.h</header>
  </customwidget>
  <customwidget>
   <class>DateInput</class>
   <extends>QLineEdit</extends>
   <header>DateInput.h</header>
  </customwidget>
  <customwidget>
   <class>QtySpinInput</class>
   <extends>QSpinBox</extends>
   <header>QtySpinInput.h</header>
  </customwidget>
  <customwidget>
   <class>ExpandableComboBox</class>
   <extends>QComboBox</extends>
   <header>ExpandableComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edit_date</tabstop>
  <tabstop>combo_batch</tabstop>
  <tabstop>spin_qty</tabstop>
  <tabstop>edit_reason_a</tabstop>
  <tabstop>edit_reason_b</tabstop>
  <tabstop>action_addnext</tabstop>
  <tabstop>action_addexit</tabstop>
  <tabstop>action_clear</tabstop>
  <tabstop>action_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
